File: Simulation/Distribution/Exponential Distribution.py
#
# Exponential Distribution Simulation
# Author: galaa
# Created on 2016/03/19 07:49:41
#

import random, math

n = int(raw_input("n = "))
Lambda = float(raw_input("Lambda = "))

while n > 0 :
    n -= 1
    print -1.0 * math.log( random.random() ) / Lambda

File: Simulation/Distribution/Exponential Distribution.cpp
/* 
 * Exponentional Distribution Simulation
 * Author: galaa
 * Created on February 25, 2011, 3:18 AM
 */

#include <cstdlib>
#include <iostream>
#include <math.h>
#include <ctime>

using namespace std;

int main()
{
    cout << "L = "; float L; cin >> L; // Parameter
    cout << "n = "; int n; cin >> n; // Sample Size
    srand(time(NULL)); // Initialize random number generator.
    for(int i=1; i<=n; i++){
        cout << -1*log((float)rand()/RAND_MAX)/L << endl;
    }
    return 0;
}

